app/controllers/posts.js

```diff
@@ -30,6 +30,9 @@ const ORDERS = [{
 }, {
     name: 'Oldest',
     value: 'published_at asc'
+}, {
+    name: 'Last Updated',
```

Comment:
```suggestion
    name: 'Recently updated',
```

Reply:
@BenJenkinson wording change requested by the design team. Once that's in place this is good to merge, thanks! 😄

Reply:
Hi @kevinansfield, I've made that wording change and brought the PR up-to-date with master.

Hopefully, it should now be good to merge!